cmake/FindDSound.cmake

```diff
@@ -8,68 +8,28 @@
 #   DSOUND_LIBRARY     - the dsound library
 #
 
-# DSOUND_INCLUDE_DIR
-file(GLOB DXSDK_INCLUDE_DIRS LIST_DIRECTORIES TRUE "C:/Program Files (x86)/Windows Kits/10/Include/*")
-if (DXSDK_INCLUDE_DIRS)
-  list(SORT DXSDK_INCLUDE_DIRS)
-  list(REVERSE DXSDK_INCLUDE_DIRS)
-endif() 
+if (WIN32)
+  include(FindWindowsSDK)
+  get_windowssdk_library_dirs(${WINDOWSSDK_PREFERRED_DIR} WINSDK_LIB_DIRS)
+  get_windowssdk_include_dirs(${WINDOWSSDK_PREFERRED_DIR} WINSDK_INCLUDE_DIRS)
```

Comment:
This looks like it should check ```WINDOWSSDK_FOUND``` before making those calls, otherwise those functions aren't defined.